### PR TITLE
close issue 1

### PR DESCRIPTION
**Pull request by codygreen**
per the Terraform GitHub issue #21260 I've changed the source_code_hash since base64sha256 now combines the file and base64sha256 functions together.

### CODE REVIEW

**Comment by jveldboom** [comment]
Thank you for taking the time to fix this!

Would you also mind updating the README to say `Requires Terraform 0.12+ to be installed.`

**Comment by codygreen** [comment]
No problem! readme updated.

File: terraform/lambda.tf [hunk 1]
```diff
@@ -3,7 +3,7 @@ resource "aws_lambda_function" "lambda" {
   function_name    = "codepipeline-teams-notifications"
   role             = "${aws_iam_role.lambda.arn}"
   handler          = "index.handler"
-  source_code_hash = "${base64sha256(file("../lambda.zip"))}"
+  source_code_hash = "${base64sha256("../lambda.zip")}"
```
Comment (jveldboom) [inline terraform/lambda.tf]: After testing this change, it actually needs to use the new `filebase64sha256` function for Terraform 0.12+. Using just `base64sha256("../lambda.zip")` will only hash the file path which will prevent Terraform from noticing a change.

```suggestion
  source_code_hash = "${filebase64sha256("../lambda.zip")}
```

**Review by jveldboom — CHANGES_REQUESTED** [review]

**Review by jveldboom — APPROVED** [review]
Awesome - thank you again for fixing this!

**Comment by codygreen** [comment]
nice catch, I have not tried modifying it past the initial deployment

**Pull request merged**